# Request to start loging signals to a file (don't include an extension)
string filename

---
# Response
bool success
string err_str
